client/packages/system/src/Vaccination/Components/VaccinationModal.tsx

```diff
@@ -87,47 +88,19 @@ export const VaccinationModal = ({
     }
   };
 
-  const alert: { severity: AlertColor; content: ReactNode } | undefined =
-    vaccination?.given
-      ? {
-          severity: 'success',
-          content: (
-            <Box display="flex" alignItems="center">
-              {t('messages.vaccination-was-given', {
-                date: localisedDate(vaccination?.vaccinationDate ?? ''),
-              })}
-              {vaccination.invoice && (
-                <Link
-                  style={{
-                    marginLeft: 6,
-                    fontWeight: 'bold',
-                    alignItems: 'center',
-                    display: 'flex',
-                  }}
-                  to={RouteBuilder.create(AppRoute.Dispensary)
-                    .addPart(AppRoute.Prescription)
-                    .addPart(vaccination.invoice.invoiceNumber.toString())
-                    .build()}
-                >
-                  {t('button.view-prescription')}
-                  <ArrowRightIcon />
-                </Link>
-              )}
-            </Box>
-          ),
-        }
-      : undefined;
+  const InfoBox = <GivenInfoBox vaccination={vaccination} />;
```

Comment:
Most of the changes in this file are just rearranging, moving out this infobox component and the batch select